update swagger guide to be compatible with openapi v3 (#2007)
port-labs/port-docs: docs/guides/all/ingest-swagger-paths-into-your-catalog.md

---

> description: Ingest API paths from an `openapi.json` file in a GitHub repository into Port

Author: ```suggestion
description: Ingest API paths from an OpenAPI spec in a GitHub repository into Port
```

displayed_sidebar: null

> title: Ingest OpenAPI paths from openapi.json file into your catalog

Author: ```suggestion
title: Ingest API paths from an OpenAPI spec in a GitHub repository into your catalog
```

---

import Tabs from "@theme/Tabs"
import TabItem from "@theme/TabItem"
import SwaggerBlueprint from '../templates/swagger/_example_swagger_blueprint.mdx'
import SwaggerWebhookConfig from '../templates/swagger/_example_swagger_webhook_config.mdx'

> # Ingest OpenAPI paths from openapi.json file into your catalog

Author: ```suggestion
# Ingest API paths from an OpenAPI spec in a GitHub repository into Port
```

> The following example shows you how to create an `apiPath` blueprint that ingests all API paths in your `openapi.json` file using Port's GitHub file ingesting feature.

Author: ```suggestion
The following example demonstrates how to create an `apiPath` blueprint, and ingest all API paths in your spec JSON file using Port's GitHub file ingesting feature.
```

Create the following blueprint definition and mapping configuration:

<details>
<summary>API path blueprint</summary>

```json showLineNumbers
{
  "identifier": "apiPath",
  "description": "This blueprint represents an OpenAPI path in our software catalog",
  "title": "API Paths",

>   "icon": "Api",

Author: no such icon, replace with an existing one
